src/main/java/org/mitre/synthea/export/BB2RIFExporter.java

```diff
@@ -1146,11 +1275,13 @@ private String capitalizeWords(String str) {
     }
   }
 
+  interface AbstractFields {}
```

Comment:
What's the purpose of this interface? If it is just a marker couldn't you use java.lang.Enum instead?

Reply:
It's just so that they have a common ancestor.  I'll try java.lang.Enum.

Reply:
refactored code based on comments

Reply:
I still don't understand this.